src/pages/homePage/apiHomePage.ts

```diff
@@ -0,0 +1,27 @@
+// api.js
+import axios from "axios";
+import { Dish } from "../../shared/interfaces/Dish";
+import CardItem from "../../shared/interfaces/CardItem";
+
+const BASE_URL = "http://localhost:3001/api";
```

Comment:
This variable need to be set as an environment variable and not here (because it's a global value that probably will be needed in other places)